testprof/rotate_test.c

```diff
@@ -204,7 +204,7 @@ int rotate_test(void)
    if (ROL64(L64, 62) != CONST64(0x04488cd115599de2)) { fprintf(stderr, "ROL64 62 failed\n"); err++; }
    if (ROL64(L64, 63) != CONST64(0x089119a22ab33bc4)) { fprintf(stderr, "ROL64 63 failed\n"); err++; }
    /* RORc */
-   if (RORc(L32, 0 ) != 0x12345678UL) { fprintf(stderr, "RORc 0  failed\n"); err++; }
+   if (RORc(L32, ze) != 0x12345678UL) { fprintf(stderr, "RORc 0  failed\n"); err++; }
```

Comment:
That's against the idea of the `RO*c()` macros! The number of bits to rotate should be a constant, not a variable.